packages/flutter/test/services/text_editing_delta_test.dart

```diff
@@ -70,6 +70,27 @@ void main() {
 
       expect(() { delta.apply(TextEditingValue.empty); }, throwsAssertionError);
     });
+
+    test('Verify TextEditingDeltaInsertion toString', () {
+      expect(
+        const TextEditingDeltaInsertion(
+          oldText: 'hello worl',
+          textInserted: 'd',
+          insertionOffset: 10,
+          selection: TextSelection.collapsed(offset: 11),
+          composing: TextRange.empty).toString(),
+        equals('TextEditingDeltaInsertion(oldText: hello worl, textInserted: d, insertionOffset: 10, selection: TextSelection.collapsed(offset: 11, affinity: TextAffinity.downstream, isDirectional: false), composing: TextRange(start: -1, end: -1))')
+      );
```

Comment:
Just my 2 cents about readability, consider formatting all calls to `expect` in this way:
```dart
    expect(
      const TextEditingDeltaInsertion(
        oldText: 'hello worl',
        textInserted: 'd',
        insertionOffset: 10,
        selection: TextSelection.collapsed(offset: 11),
        composing: TextRange.empty,
      ).toString(),
      equals(
        'TextEditingDeltaInsertion(oldText: hello worl, textInserted: d, insertionOffset: 10, selection: TextSelection.collapsed(offset: 11, affinity: TextAffinity.downstream, isDirectional: false), composing: TextRange(start: -1, end: -1))',
      ),
    );
```